turbo.json

```diff
@@ -22,6 +26,10 @@
       "inputs": ["src/**"],
       "outputs": ["dist/**"]
     },
+    "@kaizen/components#test:treeshake": {
+      "dependsOn": ["@kaizen/components#build"],
+      "inputs": ["dist/**"]
+    },
```

Comment:
Oh, this is a weird requirement

Reply:
Yeah agreed. No idea why it didn't work otherwise though. It wouldn't stop caching even when I put `cache: false` 😅